script/install.sh

```diff
@@ -231,6 +231,11 @@ parse_args() {
     local _help=
     local _activate=
 
+    # select DMD by default
+    if [[ $# -eq 0 ]] ; then
+        compiler="dmd"
+    fi
```

Comment:
Let's rather set the default value [here](https://github.com/wilzbach/installer/blob/af19e74f72ae938c353337d6be3b367a738e6be8/script/install.sh#L81).